tests/unit_tests/configuration/test_agent_config_loader.py

```
@@ -0,0 +1,218 @@
+        with patch("datus.configuration.agent_config_loader.Path.home", return_value=tmp_path / "noexist"):
+            with pytest.raises(DatusException, match="not found"):
+                parse_config_path("")
+
+
+# ---------------------------------------------------------------------------
+# ConfigurationManager
+# ---------------------------------------------------------------------------
+
+
+class TestConfigurationManager:
+    def _make_config(self, tmp_path, data: dict = None) -> Path:
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)